### ISSUE TITLE
Add guidance on shared cert extension via report

### ISSUE THREAD

**Issue by sstrassberg** [usds/SME-QA-Website#287]
After 10 day deadline for reviewing and making selections from the certificate expires, inform primary agencies that they can apply for an extension, but the cert will now begin to be shared with other interested agencies that didn't participate in the pilot.

**Comment by wslack** [comment]
To be folded into the CX after action report.

**Comment by wslack** [comment]
Could also be PR to https://smeqa.usds.gov/results/gov-wide/cx/

**Comment by stephaniefaith** [comment]
Removing my name bc i added this to the after action report. shane is still on here if he wants to add it to the PR above.

**Comment by sstrassberg** [comment]
@wslack It seems this is already covered in the "Timing" section for lessons learned under Certs:

Separately, agencies did not have enough time to select their candidate within the 10-day period. Several conversations between agencies stressed on requiring more time and communicated with OPM HR to decide how much time each agency should be given in order to select. Our recommendation in this case was to continue giving agencies a 10-day grace period with priority access. After the deadline passes, the certifications will be opened up to other agencies, noting that the original agency can apply for an extension, however, there will be other agencies now competing for selections.

So, I think we can close this issue. Thoughts?

**Comment by stephaniefaith** [comment]
Agreed!

**Comment by wslack** [comment]
Closing - handled elsewhere!
